tests/integration/replication.tcl

```diff
@@ -1236,69 +1247,80 @@ test {Kill rdb child process if its dumping RDB is not useful} {
         }
     }
 } {} {external:skip}
-
-start_server {tags {"repl external:skip"}} {
-    set master1_host [srv 0 host]
-    set master1_port [srv 0 port]
-    r set a b
-
-    start_server {} {
-        set master2 [srv 0 client]
-        set master2_host [srv 0 host]
-        set master2_port [srv 0 port]
-        # Take 10s for dumping RDB
-        $master2 debug populate 10 master2 10
-        $master2 config set rdb-key-save-delay 1000000
+foreach rdbchannel {yes no} {
```

Comment:
Whatever we decide on the naming, we should find replace all of these `rdbchannel` with rdbconn for consistency.